bitnami/cilium/templates/agent/daemonset.yaml

```diff
@@ -0,0 +1,354 @@
+        {{- end }}
+        {{- if .Values.agent.podAnnotations }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.agent.podAnnotations "context" $) | nindent 8 }}
+        {{- end }}
+      labels: {{- include "common.labels.standard" ( dict "customLabels" $podLabels "context" $ ) | nindent 8 }}
+        app.kubernetes.io/part-of: cilium
+        app.kubernetes.io/component: agent
+    spec:
+      {{- include "cilium.imagePullSecrets" . | nindent 6 }}
+      serviceAccountName: {{ template "cilium.agent.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.agent.automountServiceAccountToken }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      restartPolicy: Always
+      {{- if .Values.agent.hostAliases }}
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.agent.hostAliases "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.agent.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.agent.affinity "context" $) | nindent 8 }}
+      {{- else if not (empty .Values.agent.nodeAffinityPreset.type) }}
+      affinity:
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.agent.nodeAffinityPreset.type "key" .Values.agent.nodeAffinityPreset.key "values" .Values.agent.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.agent.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.agent.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.agent.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.agent.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.agent.priorityClassName }}
+      priorityClassName: {{ .Values.agent.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.agent.podSecurityContext.enabled }}
+      securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.agent.podSecurityContext "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.agent.terminationGracePeriodSeconds }}
+      terminationGracePeriodSeconds: {{ .Values.agent.terminationGracePeriodSeconds }}
+      {{- end }}
+      initContainers:
+        {{- include "cilium.agent.defaultInitContainers.prepareWriteDirs" . | nindent 8}}
+        {{- include "cilium.agent.defaultInitContainers.buildConfig" . | nindent 8 }}
+        {{- if .Values.agent.cgroup2.autoMount }}
+          {{- include "cilium.agent.defaultInitContainers.mountCgroup2" . | nindent 8 }}
+        {{- end }}
+        {{- if .Values.agent.bpf.autoMount }}
+          {{- include "cilium.agent.defaultInitContainers.mountBpf" . | nindent 8 }}
+        {{- end }}
+        {{- include "cilium.agent.defaultInitContainers.cleanState" . | nindent 8 }}
+        {{- if .Values.agent.waitForKubeProxy }}
```

Comment:
This is not set in values.yaml

Reply:
addressed at https://github.com/bitnami/charts/pull/26374/commits/d6b150377935aae125234d5476f88708f3d93c5e